api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -120,6 +120,7 @@ type DRPlacementControlStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:shortName=drpc
 
+// +operator-sdk:csv:customresourcedefinitions.displayName="DRPlacement Control"
```

Comment:
(fly-by comment) Why add this to this CRD only, others also seem not to have this string? Just checking if we need to add this in other places as well. (I think this has to do with existing CSV already having it for some and not for others, not sure when/why that happened)

Reply:
This is outdated now, but this change got in because I was experimenting with changing the displayName in the bundle by adding a operator-sdk marker as they call it. We have decided to get the displayName changes done using the kustomize approach and therefore these lines aren't required.